### PR TITLE
Fix broken build on go 1.12

### PR DESCRIPTION
**Pull request by grigorov**
Official repository a have https://github.com/mozilla/sops/blob/master/Makefile. make vendor is fix it build AUR package.

### CODE REVIEW

**Comment by michaelbeaumont** [comment]
This fails to build for me:

```
go.mozilla.org/sops/config imports
        github.com/mozilla-services/yaml tested by
        github.com/mozilla-services/yaml.test imports
        go.mozilla.org/yaml.v2: cannot find module providing package go.mozilla.org/yaml.v2
make: *** [Makefile:22: vendor] Error 1
==> ERROR: A failure occurred in build().
```

**Comment by grigorov** [comment]
i forgot add "go mod download" :) i'm updated PR.

**Comment by michaelbeaumont** [comment]
It still fails unfortunately. Shouldn't the `GOPATH` also be modified for these commands like it is in the existing command?
Can I ask why you need to support Go 1.12?
Is there a way instead to make 1.12 behave like 1.13?
What is there error with 1.12?

**Comment by grigorov** [comment]
I will use arch based distrib (Manjaro) with stable repo. Go package is version: 1.12.9 in community. If I use your AUR, I can't build sops, error of find Golang modules. I use method from official repository sops for build go 1.12... Maybe go 1.13 all changes.

File: sops/PKGBUILD [hunk 1]
```diff
@@ -19,6 +19,8 @@ prepare() {
 
 build() {
   cd src/go.mozilla.org/sops
+  go mod download
+  make vendor
   env GOPATH="${srcdir}" go build \
```
Comment (michaelbeaumont) [inline sops/PKGBUILD]:
```suggestion
   env GO111MODULE=on GOPATH="${srcdir}" go build \
```
Can you try that? (without your other changes please)
Sops switched to using modules for this release

Comment (grigorov) [reply sops/PKGBUILD]: It's work. I updated PR

**Comment by michaelbeaumont** [comment]
Updated in the AUR!

Thanks @grigorov!

**Pull request merged**